src/main/java/sg/ncl/testbed_interface/TeamPageApplyTeamForm.java

```diff
@@ -20,10 +20,20 @@
     private String teamWebsite = "http://default.com";
 
     private String teamOrganizationType;
+
+    private String joinTeamReason;
     
     @NotNull(message="Please select one of them")
     private String isPublic = TeamVisibility.PUBLIC.toString();
-    
+
+    public String getJoinTeamReason() {
+        return joinTeamReason;
+    }
+
+    public void setJoinTeamReason(String joinTeamReason) {
+        this.joinTeamReason = joinTeamReason;
+    }
+
     public TeamPageApplyTeamForm() {   
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)